ansible/files/telemetry.sh

```diff
@@ -67,6 +67,24 @@ sudo sh -c "sudo cat <<EOF > /etc/presto/conf/jvm.config
 EOF"
 sudo pkill presto
 
+# upgrade hive
+tmp="$(mktemp -d)"
+cd "$tmp"
+aws s3 sync s3://telemetry-presto-emr/packages/ .
+tar zxvf apache-hive-1.2.1-bin.tar.gz
```

Comment:
I've updated the code. Of note though, this will not make the Hive 1.3 upgrade much easier (that's the version I originally tried to update to) as there are some bigger configuration/compatibility changes that need to be made to support that.